Projects/Data/Sources/Entities/Resolutions.swift

```diff
@@ -0,0 +1,12 @@
+//
+// Created by Igor Tarasenko on 2019-02-07.
+// Licensed under the MIT license
+//
+
+import Foundation
+
+public struct Resolution: Codable {
```

Comment:
The file is named `Resolutions`, yet the only type it defines is `Resolution`. Typo or are we planning to define more types here?

Reply:
Fixed